_layouts/default.html

```diff
@@ -50,6 +50,7 @@
                 <li><a href="pdf_manager_batch.html">PDF Manager Batch</a></li>
                 <li><a href="pdf_manager_zip_file.html">PDF Manager Zip File</a></li>
                 <li><a href="program.html">Program</a></li>
+                <li><a href="social_security_numbers.html">Social Security Numbers <span class="label label-warning">Prototype</span></a></li>
```

Comment:
That's what we did for every single link with the "prototype" label, I don't know if it will change anything in the UI or if there's a good reason to do it like that.